sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeHealthcareEntitiesOptions.java

```diff
@@ -40,6 +40,20 @@ public AnalyzeHealthcareEntitiesOptions setIncludeStatistics(boolean includeStat
         return this;
     }
 
+    /**
+     * Set the value of {@code disableServiceLogs}.
+     *
+     * @param disableServiceLogs If a boolean value was specified in the request this field will contain
+     * logging information about the document payload.
+     *
+     * @return the {@link AnalyzeHealthcareEntitiesOptions} object itself.
+     */
+    @Override
+    public AnalyzeHealthcareEntitiesOptions setServiceLogsDisabled(boolean disableServiceLogs) {
```

Comment:
this reads kind of weird, is `SetDisableServiceLogs` bad English? or what was the reason for the change?

Reply:
I think `SetDisableServiceLogs`  sounds weird. `disabled` is a adjective. so `Set---Disabled` more sounds like english.

Reply:
hmm I guess I am not the right person to ask as I am non-English native. Better to ask someone else :P

Reply:
Asked Jonathan about it, he suggested to go with `setServiceLogsDisabled`